app/components/m-header.hbs

```diff
@@ -58,6 +58,16 @@
       {{/if}}
     </Auk::Toolbar::Group>
     <Auk::Toolbar::Group @position="right" class="auk-u-mr-3">
+      {{#if
+           (and this.currentSession.impersonatedRole
+           (not-eq this.currentSession.role this.currentSession.impersonatedRole))
```

Comment:
Provide a `get isImpersonated()` on the `currentSession` service containing this logic to 'hide' service internals